outputs.tf

```
@@ -1,4 +1,4 @@
 output "bucket" {
   description = "AWS S3 Bucket object"
-  value       = length(aws_s3_bucket.this) > 0 ? aws_s3_bucket.this[0] : null
+  value       = length(aws_s3_bucket.this) > 0 ? aws_s3_bucket.this : null
```

since we no longer have a count on the resource, we can simplify further:

```suggestion
  value       = aws_s3_bucket.this
```